### PR TITLE
Show Transaction Hash of Ledger in MultiSig Toolkit

### PR DESCRIPTION
**Pull request by jnaulty**
## Description 

Show the Transaction Hash the user sees on the Ledger as an option for the User in the Multisig Toolkit.

Currently, any user signing a Sui transaction with a ledger has a hard time verifying the transaction they are signing is the transaction they expect to sign.
This feature provides a way for the user to verify the Transaction Hash presented by the Ledger from the MultiSig Toolkit Interface

## Test plan 

Tested Locally

created transaction:
```
sui client transfer-sui --to 0x497f1b4197a6d9f7db028d79d068af120cf37d0250acd392eedb312a537712f9 --sui-coin-object-id 0xa9692324f4fd9ea6771e57aa0c600d18852da022d4bfc1f32a7e9a0c99b7eef9 --gas-budget 300000000 --serialize-unsigned-transaction
[warn] Client/Server api version mismatch, client api version : 1.34.0, server api version : 1.36.0
AAABACBJfxtBl6bZ99sCjXnQaK8SDPN9AlCs05Lu2zEqU3cS+QEBAQABAABJfxtBl6bZ99sCjXnQaK8SDPN9AlCs05Lu2zEqU3cS+QGpaSMk9P2epnceV6oMYA0YhS2gItS/wfMqfpoMmbfu+QUAAAAAAAAAIPBDd5NqCOpDQdJEBJ8Vwn9iC+AVUxn/Z8opYbvVTYgESX8bQZem2ffbAo150GivEgzzfQJQrNOS7tsxKlN3EvnoAwAAAAAAAACj4REAAAAAAA==

```

Compute Transaction hash in multisig toolkit: `0x46ac3b8e4f61981f7c016e0bcb53cc2bbd4bc52f8d5e51e335e0a5dc15ab9cb5`

image

Verify same hash Ledger shows

image

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:

### CODE REVIEW

File: dapps/multisig-toolkit/src/routes/offline-signer.tsx [hunk 1]
```diff
@@ -70,6 +73,24 @@ export default function OfflineSigner() {
...
+	const computeLedgerTransactionHash = async () => {
+		try {
+			// Decode the base64-encoded transaction bytes
+			const decodedBytes = fromBase64(bytes);
+			const intentMessage = messageWithIntent('TransactionData', decodedBytes);
+			const intentMessageDigest = blake2b(intentMessage, { dkLen: 32 });
+			const intentMessageDigestHex = toHex(intentMessageDigest);
+			setLedgerTransactionhash(intentMessageDigestHex);
+		} catch (error) {
+			console.error('Error computing blake2b hash:', error);
```
Comment (manolisliolios) [inline dapps/multisig-toolkit/src/routes/offline-signer.tsx:89]: We can skip the console error here

**Review by manolisliolios — APPROVED** [review]
🚀

File: dapps/multisig-toolkit/src/routes/offline-signer.tsx (hunk 1 above)
Comment (jnaulty) [reply dapps/multisig-toolkit/src/routes/offline-signer.tsx]: ✅  done

**Review by hayes-mysten — APPROVED** [review]
Looks good (other than the blake2b link in package.json)

**Pull request merged**